### PR TITLE
FIX brazilian portuguese translation

### PR DESCRIPTION
**Pull request by raphaeldealmeida**
FIX some wrong terms on translation

### CODE REVIEW

File: adminer/lang/pt-br.inc.php [hunk 1]
```diff
@@ -66,19 +66,19 @@
...
+	'Unable to select the table' => 'Não é possível selecionar a Tabela',
+	'Invalid CSRF token. Send the form again.' => 'Token CSRF inválido. Enviar o formulário novamente.',
 	'Comment' => 'Comentário',
 	'Default values' => 'Valores padrões',
 	'%d byte(s)' => array('%d byte', '%d bytes'),
 	'No commands to execute.' => 'Nenhum comando para executar.',
 	'Unable to upload a file.' => 'Não é possível enviar o arquivo.',
 	'File upload' => 'Importar arquivo',
-	'File uploads are disabled.' => 'Importação de arquivos desablilitada.',
+	'File uploads are disabled.' => 'Importação de arquivos desabilitada.',
```
Comment (Jeckerson) [inline adminer/lang/pt-br.inc.php]:
```suggestion
	'File uploads are disabled.' => 'Importação de arquivos estão desligados.',
```

Comment (raphaeldealmeida) [reply adminer/lang/pt-br.inc.php]: 'Importação de arquivos estão **desligados**.' It is wrong for brazilian portuguese speakers.
alternatives:
> Importação de arquivos **está desligada**

> Importação de arquivos **desablilitada**  (it's is my choise)

Comment (Jeckerson) [reply adminer/lang/pt-br.inc.php]: Well, **está desligada** also make sense.

Meanwhile the word **desablilitada** can be substituted with **desactivada.** Which is more generic, IMO.

Comment (raphaeldealmeida) [reply adminer/lang/pt-br.inc.php]: word **desactivada** don't exists in brazilian portuguese.

Comment (Jeckerson) [reply adminer/lang/pt-br.inc.php]: 1. https://www.dicio.com.br/desativada/
2. https://www.dicionarioinformal.com.br/desativado/

**Comment by raphaeldealmeida** [comment]
'desa**c**tivada' isn't same 'desativada', but ok.
I think you need to edit pt.inc.php file

**Pull request merged**